test/lib/adapters/Memory_write.js

```diff
@@ -25,11 +24,9 @@ test("glob resources from application.a w/o virtual base path prefix", async (t)
 	const res = createResource({
 		path: "/app/index.html"
 	});
-	await dest.write(res)
-		.then(() => dest.byGlob("/**/*.html"))
-		.then((resources) => {
-			t.is(resources.length, 1, "Found exactly one resource");
-		});
+	await dest.write(res);
+	const resources = await dest.byGlob("/**/*.html");
+	t.is(resources.length, 1, "Found exactly one resource");
```

Comment:
call setContent to the written resource and check that original resource is not changed

Reply:
done